backend/tests/e2e/test_e2e_chat.py

```
@@ -0,0 +1,77 @@
+import pytest
```

Ruff (0.11.9)

`pytest` imported but unused

Remove unused import: `pytest`

(F401)